common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -557,6 +559,82 @@ std::vector<uint8_t> wireless_utils::calc_5g_20MHz_subband_channels(
     return channels;
 }
 
+void wireless_utils::get_channel_preferences(
+    beerocks::message::sWifiChannel supported_channels[],
+    std::map<beerocks::message::sPreference, std::list<beerocks::message::sWifiChannel>>
+        &preferences)
+{
+    std::list<uint8_t> oper_classes;
+    wireless_utils::get_supported_operating_classes(supported_channels, oper_classes);
```

Comment:
There's no need to do get_supported_operating_classes first, it is implied by the matching in the loop below. The loop below can directly iterate over all operating classes.

Reply:
Done.